tiff_2_png/tiff_2_png_converter.py

```diff
@@ -206,8 +214,13 @@ def convert_to_png(
                     ## check all images have the same data type
                     if bits_per_sample == "":
                         bits_per_sample = img.dtype
-                        global_max_pixel = np.max(img)
-                        global_min_pixel = np.min(img)
+                        if img.dtype == np.float32:
```

Comment:
since it's saved above might as well use it here. 
```suggestion
                        if bits_per_sample == np.float32:
```

Reply:
I removed the float32 conversion logic. It doesnt apply to any of our cases. I'll create an issue to add this as a new feature